frontend/src/models/models.js

```diff
@@ -26,12 +26,22 @@ module.exports = app => app.component('models', {
     shouldShowExportModal: false,
     shouldExport: {},
     sortBy: {},
-    query: {}
+    query: {},
+    scrollHeight: 0,
+    limit: 50,
+    interval: null
   }),
+  watch: {
+
+  },
   created() {
     this.currentModel = this.model;
   },
+  beforeDestroy() {
+    document.removeEventListener('scroll', this.onScroll, true);
+  },
   async mounted() {
+    document.addEventListener('scroll', this.onScroll, true);
```

Comment:
The reason why you're getting the "lost container" error is because `this` won't be the Vue component in `this.onScroll()`. You need to do `() => this.onScroll()` to retain context.